include/zephyr/drivers/misc/renesas_lvd/renesas_lvd.h

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2025 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Public APIs for the Renesas LVD voltage monitor
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_MISC_RENESAS_LVD_H_
+#define ZEPHYR_INCLUDE_DRIVERS_MISC_RENESAS_LVD_H_
+
+/**
+ * @brief Renesas LVD driver public APIs
+ * @defgroup renesas_lvd_interface Renesas LVD driver APIs
+ * @ingroup misc_interfaces
+ * @{
+ */
+
+#include <stdint.h>
+#include <zephyr/sys/slist.h>
+#include <zephyr/device.h>
+#include <zephyr/kernel.h>
+#include <zephyr/internal/syscall_handler.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief LVD voltage position relative to threshold.
+ *
+ *  This enum indicates whether the monitored voltage is
+ *  currently above or below the configured threshold.
+ */
+typedef enum {
+	/** Voltage is above threshold. */
+	RENESAS_LVD_POSITION_ABOVE = 0,
+
+	/** Voltage is below threshold. */
+	RENESAS_LVD_POSITION_BELOW,
+} renesas_lvd_position_t;
```

Comment:
This maps well to https://github.com/zephyrproject-rtos/zephyr/blob/90723c86ed7f4166bb94cbf2c65f1e127e51977d/include/zephyr/drivers/comparator.h#L60-L74